contrib/ci-builders/tekton/tekton-labs/releases/tekton-pipeline-v0.37.0.yaml

```diff
@@ -1501,7 +1501,7 @@ data:
     # default-managed-by-label-value contains the default value given to the
     # "app.kubernetes.io/managed-by" label applied to all Pods created for
     # TaskRuns. If a user's requested TaskRun specifies another value for this
-    # label, the user's request supercedes.
+    # label, the user's request supersedes.
```

Comment:
"supercedes" is just the British English spelling. We don't normally worry about which variant of English is used in comments.

Reply:
Apparently not specifically British, but a common spelling for this word nonetheless. It comes from the Middle French _superceder_, which is good enough for me.